extensions/wikia/ThemeDesigner/ThemeDesignerHelper.class.php

```diff
@@ -4,13 +4,15 @@ class ThemeDesignerHelper {
 
 	public static function checkAccess() {
 		global $wgUser;
+
 		// @FIXME when we're out of beta editinterface needs to be removed and themedesgner set to true for sysops
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)